### PR TITLE
faster make test

### PR DESCRIPTION
**Pull request by orichters**
## Purpose of this PR

- quicker tests (~ 13 min instead of 22 min)
- instead of running a testOneRegi with `action = c`, run `Rscript start.R --gamscompile`. Is somehow much faster (2 min -> 30 seconds)
- don't compile 27 AMT runs, but just a selection of 6 that should cover the modules that are used. Reduces runtime from 12 min to < 5 min. I added a EU21 run as requested below. It is time-consuming as new input data has to be downloaded and distributed twice (before and after the run)

## Type of change

- [x] Refactoring

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [ ] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

### CODE REVIEW

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — CHANGES_REQUESTED** [review]
> * Adding a EU21 run is maybe advisable, but also time-consuming as new input data has to be downloaded and distributed twice (before and after the run)

Definitely test `EU21`.  Download times are not an issue on the cluster, people can always test there, or cache locally if they are so inclined.
But changes in regional aggregation are a much more serious source of errors then different module realisations, in my experience.

File: start_bundle_coupled.R [hunk 1]
```diff
@@ -183,11 +183,6 @@ if (any(! file.exists(c(path_settings_coupled, path_settings_remind))) ||
   stop("Missing files or directories, see in red above.")
 }
 
-if ("--gamscompile" %in% flags && ! file.exists("input/source_files.log")) {
```
Comment (orichters) [inline start_bundle_coupled.R:186]: that is not needed anymore, as `runGamsCompile` does that alone

**Comment by orichters** [comment]
Something went wrong with the rebase, not fixing it now as #1539 will lead to merge conflicts anyway so that has to be redone anyway.

**Comment by orichters** [comment]
Ok, including the EU21 run, time is down from ~22 to 14 min.

**Comment by orichters** [comment]
@0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q: Thanks for the comment, I added SSP2EU-EU21-Base to the list of scenarios to be compiled, maybe you can remove "changes requested".

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
Sure.  I thought you planned on making a new merge request in order to not deal with the conflicts.

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]

**Comment by orichters** [comment]
> Sure. I thought you planned on making a new merge request in order to not deal with the conflicts.

I just wanted to wait for #1536 and #1539 to be merged to avoid fixing the conflicts several times.

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**